### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by dhoman01**
Ready to finalize UI and demo.

### CODE REVIEW

File: api/test/unit/controllers/AffiliateController.test.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        }
+      },
+      param: function(){
+        return 'aaaaaaaaaaa';
+      }
+    }
+    res = {
+      ok: sinon.spy()
+    }
+    next = sinon.spy();
```
Comment (sysMachinist) [inline api/test/unit/controllers/AffiliateController.test.js]: Hello Dustin

Comment (sysMachinist) [inline api/test/unit/controllers/AffiliateController.test.js]: !resolved

File: api/controllers/AuthController.js [hunk 2]
```diff
@@ -9,10 +9,13 @@
  */
 
 module.exports = require('waterlock').waterlocked({
-  /* e.g.
-    action: function(req, res){
-  
+
```
Comment (dhoman01) [inline api/controllers/AuthController.js]: Needs inline documentation to explain purpose. !fix

File: api/controllers/UserController.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ module.exports = require('waterlock').actions.user({
     Salesforce.retrieveAsync({sObject: "Contact", sfIds: req.session.user.sfId}).bind({})
```
Comment (dhoman01) [inline api/controllers/UserController.js]: Needs inline documentation! !fix

File: api/test/unit/controllers/AffiliateController.test.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
+var _ = require('lodash');
```
Comment (dhoman01) [inline api/test/unit/controllers/AffiliateController.test.js]: Comment purpose of imported dependencies. !fix

File: assets/js/controllers/LoginController.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+(function(){
```
Comment (dhoman01) [inline assets/js/controllers/LoginController.js]: Needs inline documentation!

**Pull request merged**